index.html

```diff
@@ -40,5 +40,209 @@ <h1>I'm Benjamin Glad to see you!</h1>
             </div>
         </section>
     </div>
+    
+    <section class="middle-section-container">
+        <div class="image-area">
+          <img src="photo/logos/Snapshoot-Portfolio.png" alt="">
+        </div>
+
+      <div class="hero">
+          <div class="tonic">
+              <h1>Tonic</h1>
+            <div class="canopy-wrapper">
+              <p>CANOPY</p>
+              <ul>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+          </div>
+
+          <div class="text-area">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+          </div>
+        </div>
+
+        <div class="middle-section-nav-bars">
+          <ul class="technology">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+
+        <div class="project-link">
+          <h1>See Project</h1>
+        </div>
```

Comment:
- [ ] As I can see here that you didn't add  the button styling: Active, Hover and Pressed please follow the styling and add them to your button to match the Figma

- [ ] I think it would be better if you wrap the button in `a` tag because in the future you will need to put a real project so the recruiters can see your project well 👌🏻👌🏻💯